### ISSUE TITLE
Ability to graph count of strings

### ISSUE THREAD

**Issue by adamantium8** [bluenote10/PandasDataFrameGUI#12]
Hi bluenote10,

Really great project but I wanted to know if you could possibly add the ability to graph counts of strings, when I currently try to display a str.dtype column, python throws up an error. I was hoping it could display it in an alphabetical order or the order its aggregated in (similar to the image below).

![image](https://user-images.githubusercontent.com/30756449/28997778-b875e1be-7a14-11e7-8359-2b1099c8445c.png)

Thanks,
adamantium8

**Comment by bluenote10** [comment]
Good point, I have just pushed a small change which allows to plot like this:

![image](https://user-images.githubusercontent.com/3620703/29184373-40a17ffe-7e06-11e7-910a-60839eb415df.png)

Does that work for you?

**Comment by arsalan31** [comment]
Thanks bluenote10,

Almost, I'm getting some errors from certain str columns (this maybe due to some python2 vs python3 issue), the column doesn't have any float values.
![image](https://user-images.githubusercontent.com/22502670/29274406-baed9454-80ff-11e7-843d-6cad80bf09dd.png)

While I'm at it, I'm just going to mention a few other issues.
Also a minor issue with the names not resizing.
![image](https://user-images.githubusercontent.com/22502670/29273845-af78913e-80fd-11e7-827a-421dc2dd83b1.png)

Similar to the above, the datetime axis isn't readjusting in the histogram
![image](https://user-images.githubusercontent.com/22502670/29274287-3b81809a-80ff-11e7-9cdf-26e0624fdd8e.png)

And finally right clicking the main Data Frame table gives me an error 
![image](https://user-images.githubusercontent.com/22502670/29274361-83ebce62-80ff-11e7-9e14-a1f5d726e628.png)
